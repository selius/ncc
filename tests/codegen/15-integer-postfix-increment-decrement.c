int main()
{
	int a;
	int b;

	a = 4;

	b = a++;
	__print_int(a);
	__print_int(b);

	a++;
	b = a++;
	__print_int(a);
	__print_int(b);

	a = -2;

	b = a--;
	__print_int(a);
	__print_int(b);

	a--;
	b = a--;
	__print_int(a);
	__print_int(b);

	return 0;
}
