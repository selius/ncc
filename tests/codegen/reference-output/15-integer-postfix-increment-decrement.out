5
4
7
6
-3
-2
-5
-4
